File: app/src/main/java/net/emojiparty/android/jishotomo/data/room/SenseFts.java
package net.emojiparty.android.jishotomo.data.room;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.Fts4;

@Fts4(contentEntity = Sense.class)
@Entity(tableName = "sensesFts")
public class SenseFts {
  @NonNull
  private String glosses;

  public SenseFts(String glosses) {
    this.glosses = glosses;
  }

  public String getGlosses() {
    return glosses;
  }
}

File: app/src/main/java/net/emojiparty/android/jishotomo/data/room/SenseFts.kt
package net.emojiparty.android.jishotomo.data.room

import androidx.room.Entity
import androidx.room.Fts4

@Fts4(contentEntity = Sense::class)
@Entity(tableName = "sensesFts")
data class SenseFts(val glosses: String)
